test/evergreen.yml

```diff
@@ -4238,30 +4083,20 @@ buildvariants:
 
 - name: cppsuite-stress-tests
   display_name: "Cppsuite Stress Tests"
-  batchtime: 960 # Twice a day
+  batchtime: 480 # 3 times a day
   run_on:
-  - ubuntu2004-medium
+  - ubuntu2004-test
   expansions:
     test_env_vars:
       WT_TOPDIR=$(git rev-parse --show-toplevel)
       WT_BUILDDIR=$WT_TOPDIR/cmake_build
       LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_TOPDIR/TCMALLOC_LIB/lib
     cmake_generator: Ninja
     make_command: ninja
-    posix_configure_flags:
-      -DHAVE_DIAGNOSTIC=1
-      -DENABLE_PYTHON=1
-      -DENABLE_SNAPPY=1
-      -DENABLE_STRICT=1
-      -DENABLE_STATIC=1
-      -DENABLE_TCMALLOC=1
-      -DENABLE_ZLIB=1
-      -DCMAKE_CXX_FLAGS="-ggdb"
-      -DCMAKE_PREFIX_PATH="$(pwd)/../TCMALLOC_LIB"
+    CMAKE_PREFIX_PATH: -DCMAKE_PREFIX_PATH="$(pwd)/../TCMALLOC_LIB"
```

Comment:
I know you weren't convinced about the TCMalloc stuff, but it looks OK? Sorry that the issue (if any) isn't jumping out at me.

Reply:
In the patch build I'm not convinced we pick up that we have TCMALLOC_LIB. That is something I need to confirm.

Reply:
OK, if you want a hand debugging then shout out.

Reply:
Found it, was based on the directory when the var was resolved. Fixed!

Reply:
Nice!